WordPress/Classes/ViewRelated/Site Creation/DesignSelection/SiteDesignContentCollectionViewController.swift

```diff
@@ -114,7 +117,9 @@ class SiteDesignContentCollectionViewController: CollapsableHeaderViewController
     }
 
     private func handleError(_ error: Error) {
-        /* ToDo */
+        let titleText = NSLocalizedString("Unable to load this content right now.", comment: "Informing the user that a network request failed becuase the device wasn't able to establish a network connection.")
+        let subtitleText = NSLocalizedString("Check your network connection and try again or create a blank page.", comment: "Default subtitle for no-results when there is no connection with a prompt to create a new page instead.")
```

Comment:
Do we want this to say "or create a blank page"?

Reply:
Oops probably not haha